### PR TITLE
enhance: Port Pouch Cli to Darwin(MacOS)

### PR DESCRIPTION
**Pull request by xuzhenglun**
### Ⅰ. Describe what this PR did
Make Pouch command line tool work on Darwin platform(MacOS).

### Ⅲ. Describe how you did it
fix missing syscall on MacOS and remove dependence between ctrd and cli.

### Ⅳ. Describe how to verify it
compiled and run well.

### CODE REVIEW

File: apis/types/image_progress_info.go [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package types
+
+import "time"
+
+// ProgressInfo represents the status of downloading image.
+type ProgressInfo struct {
```
Comment (allencloud) [inline apis/types/image_progress_info.go]: In dir /apis/types, all structs are auto generated by apis/swagger.yml. So if you could wanna add a struct here, you must edit the swagger.yml and auto generate this struct. Thanks. And you can refer to more details here at https://github.com/alibaba/pouch/blob/master/apis/README.md

**Comment by allencloud** [comment]
This seems quite useful for pouchcontainer, and could you help to review this? @fuweid

**Comment by fuweid** [comment]
LGTM

**Comment by fuweid** [comment]
Thanks @xuzhenglun !

**Pull request merged**